tests/digest.bats

```diff
@@ -29,35 +29,6 @@ fromreftest() {
   rm -fr ${pushdir}
 }
 
-@test "from-by-digest-s1" {
```

Comment:
Welp. Docker registry image no longer supports v1 schema. Code has been excised. There is no way to push a v2s1 image to this registry. Is it OK to just remove all this v1 testing?